packages/core/src/components/badge/badge.tsx

```diff
@@ -15,10 +15,18 @@ export class TdsBadge {
   /** Sets component size. */
   @Prop() size: 'lg' | 'sm' = 'lg';
 
+  /** Defines aria-live attribute */
+  @Prop() liveAnnounce: 'off' | 'polite' | 'assertive' = 'polite';
```

Comment:
Since we had a case in Accordion Item where it was not possible to use ariaLevel, I wonder can we follow convention here and have it named ariaLiveValue instead?